lib/lynis/lynistest.pm

```diff
@@ -205,6 +206,52 @@ sub parse_lynis_section_content {
     return @section_content;
 }
 
+sub check_exceptions {
```

Comment:
LGTM, only minor comment here,
could you please add some description why so many 'exceptions' here?

Reply:
sure thing!